test/ObservableQueue-spec.js

```diff
@@ -76,10 +76,13 @@ describe('ObservableQueue', function () {
 
     const queue = new ObservableQueue({ entityType, swimlane, eventHandlers });
 
-    events.forEach((event) => {
-      new Promise((resolve, reject) => {
-        queue.queueEvent({ event, resolve, reject });
-      });
+    Promise.all(events.map((event) =>
```

Comment:
Solves this:

> UnhandledPromiseRejectionWarning :(
 https://gist.github.com/dartvandru/5c7fc23415d22ce4557b22eac7971e8f